plugins/reference-assets/core/src/assets/choice/types.ts

```diff
@@ -0,0 +1,73 @@
+import type {
+  Asset,
+  AssetWrapper,
+  Binding,
+  ValidationResponse,
+  Schema,
+} from "@player-ui/player";
+import type { BeaconDataType } from "@player-ui/beacon-plugin";
+
+/**
+ * User actions can be represented in several places.
+ * Each view typically has one or more actions that allow the user to navigate away from that view.
+ * In addition, several asset types can have actions that apply to that asset only.
+ */
+export interface ChoiceAsset<AnyTextAsset extends Asset = Asset>
+  extends Asset<"choice"> {
+  /** A text-like asset for the choice's label */
+  title?: AssetWrapper<AnyTextAsset>;
+
+  /** Asset container for a note. */
+  note?: AssetWrapper<AnyTextAsset>;
+
+  /** The location in the data-model to store the data */
+  binding?: Binding;
+
+  items?: Array<ChoiceItem>;
+
+  /** Additional optional data to assist when a choice item is selected */
+  metaData?: {
+    /** Additional data to beacon */
+    beacon?: BeaconDataType;
+  };
+}
+
+type ValueType = string | number | boolean | undefined | null;
```

Comment:
1. There might be portability issues if this type isn't exported
2. `undefined` can probably be excluded from the union if  `value` is made optional on both authored and transformed types